Source/WPEFramework/Config.h

```diff
@@ -400,6 +401,7 @@ namespace PluginHost {
                 Add(_T("volatilepath"), &VolatilePath);
                 Add(_T("proxystubpath"), &ProxyStubPath);
                 Add(_T("postmortempath"), &PostMortemPath);
+                Add(_T("messagecontrolpath"), &MessageControlPath);
```

Comment:
Let's add this to the messaging section so that we group all messaging settings, then it is clear they all belong together.
(and then  you don't have to forward these settings seperately in the Open of tghe MessageControlUnit)
And as we are at this let's also move messaging port to that section (it is now in line 423), it was an oversight it was not moved before :) (and btw you don;t have to worry that it lead to backwards compatibility problems as messagingport is only applicable to windows)

Reply:
moved both to messaging config